code/OAuthServerController.php

```diff
@@ -83,11 +115,10 @@ public function __construct()
             $this->myRepositories['client'],
             $this->myRepositories['accessToken'],
             $this->myRepositories['scope'],
-            $privateKey,
-            $encryptionKey
+            $this->privateKey,
```

Comment:
`$this->server = @new AuthorizationServer(` the `@` should be removed, this is is part of the module core code, but it is a bad practise.